.github/workflows/cdelivery-ecs-backend.yml

```diff
@@ -0,0 +1,272 @@
+name: CDelivery reusable workflow for ECS
+
+# Control when the action will run.
+on:
+  # Allow the workflow to be reusable
+  workflow_call:
+    inputs:
+      ecs-task-definition:
+        required: true
+        type: string
+    secrets:
+      aws-access-key-id:
+        required: true
+      aws-secret-access-key:
+        required: true
+      aws-region:
+        required: true           
+      ecs-cluster:
+        required: true
+      ecs-service:
+        required: true
+      ecr-repository:
+        required: true
+      container-name:
+        required: true
+      container-name-2:
+        required: true
+      container-name-3:
+        required: true
+      container-image-2:
+        required: true
+      container-image-3:
+        required: true
+      apps-jwt-secret:
+        required: true
+      apps-plugin:
+        required: true
+      auth-client-host:
+        required: true
+      auth-token-expiration-in-minutes:
+        required: true
+      auth-token-jwt-secret:
+        required: true
+      avatars-path-prefix:
+        required: true
+      chatbox-plugin:
+        required: true
+      client-host:
+        required: true
+      cookie-domain:
+        required: true
+      cors-origin-regex:
+        required: true
+      database-logs:
+        required: true
+      email-links-host:
+        required: true
+      embedded-link-item-iframely-href-origin:
+        required: true
+      embedded-link-item-plugin:
+        required: true
+      file-storage-root-path:
+        required: true
+      files-path-prefix:
+        required: true
+      hidden-tag-id:
+        required: true
+      hostname:
+        required: true
+      image-classifier-api:
+        required: true
+      jwt-secret:
+        required: true
+      login-item-tag-id:
+        required: true
+      mailer-config-from-email:
+        required: true
+      mailer-config-password:
+        required: true
+      mailer-config-smtp-host:
+        required: true
+      mailer-config-username:
+        required: true
+      node-env:
+        required: true
+      node-env-2:
```

Comment:
Here each of them refers to a different container (1 and 2). I can change it to node-env-container-2 if you think that's okay, so that way is more explicit.